plugins/msr/README.md

```diff
@@ -0,0 +1,14 @@
+MSR
+===
+
+Introduction
+------------
+
+This plugin checks if the MSRs indicate the Direct Connect Interface (DCI) is
```

Comment:
Probably need to describe what MSR actually is in this readme or link to something explaining it.